src/equations/acoustic_perturbation_2d.jl

```diff
@@ -344,8 +346,9 @@ end
                                                               equations::AcousticPerturbationEquations2D)
     λ = dissipation.max_abs_speed(u_ll, u_rr, orientation_or_normal_direction,
                                   equations)
-    diss = -0.5 * λ * (u_rr - u_ll)
-    z = zero(eltype(u_ll))
+    diss = -0.5f0 * λ * (u_rr - u_ll)
+    z = 0
```

Comment:
Since we are getting started with this process here, it would be useful to document some guidelines for users & developers somewhere, such that they can do The Right Thing going forward and either write `xxf0` or `convert(RealT, xx)`. Optimally, this should also be mentioned in the review guidelines.

It does not have to be in this PR, but I recommend to do it soon to get support from everyone.